ractor.rb

```diff
@@ -138,8 +146,13 @@
 # Notice that even +inspect+ (and more basic methods like <tt>__id__</tt>) is inaccessible
 # on a moved object.
 #
-# Besides frozen objects, shareable objects are instances of Module, Class and Ractor itself. Modules
-# and classes, though, can not access instance variables from ractors other than main:
+# Besides frozen objects, there are shareable objects. Class and Module objects are shareable so
```

Comment:
There are 3 types:

* immutable (frozen and refer to sharable objects)
* classes/modules
* special shareable
  * Ractor
  * Lvar/TVar/... (will be added by core or extension)